content/faq/regtest-setup-guide.md

```diff
@@ -3,156 +3,188 @@ title: How do I setup an isolated regtest network to test LBRY?
 category: developer
 ---
 
-## Prerequisites
+## Why use a regtest server
+A regtest server provides for a way to instantly generate blocks so that transactions can be instantaneous, so ultimately no waiting for confirmations from the blockchain. Also, no problem if you accidently corrupt your wallet, since no real funds are lost! Just delete the files and setup a new one.
+
+## Setup
 
 To begin setting up the network, there are a few things you need.
 
-Note that you need a Linux distribution to run all this. A virtual machine is
-fine.
+Note that you need a Linux distribution to run all this. A virtual machine is fine.
```

Comment:
should also work on Mac , 
Can you also state here what distribution you used to test this instruction. I.e, 
"These instructions specifically were tested on Ubuntu version XX.XX "